tests/onnx/conftest.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright (c) 2022 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+
+def pytest_addoption(parser):
```

Comment:
Some parameters that doing the same role in tf or torch may be named in a unified way, such as `--data` in tf or torch vs `--data-dir` in onnx.

Reply:
It's a good point. I think it would be nice to keep the similar argument structure for every framework testing as possible. But, I think `--data-dir` or `--data-root` is more appropriate than `--data` because it is actually directory path. @dongkwan-kim01 would you please make a backlog ticket in our JIRA about this?

Reply:
Yes, I'll make the ticket for it. Also agree with the `--data-dir` is more intuitive for the directory path.